From 807c966d15651ddebb0e8a616e01750e3f031faf Mon Sep 17 00:00:00 2001
From: Mohammad Rafi Shaik <quic_mohs@quicinc.com>
Date: Mon, 27 Jan 2025 16:01:45 +0530
Subject: [PATCH 1/8] PENDING: ASoC: dt-bindings: q6dsp: Add High Speed Audio
 DAI ID support for MI2S and TDM

Add High speed audio DAI ID support for MI2S and TDM.

Change-Id: I490b9a85c96fd9d5f2922ce91360c4a7c2411ceb
Signed-off-by: Mohammad Rafi Shaik <quic_mohs@quicinc.com>
Upstream-Status: Pending
---
 .../sound/qcom,q6dsp-lpass-ports.h            | 20 +++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/include/dt-bindings/sound/qcom,q6dsp-lpass-ports.h b/include/dt-bindings/sound/qcom,q6dsp-lpass-ports.h
index 39f203256c4f..8686aba8ea24 100644
--- a/include/dt-bindings/sound/qcom,q6dsp-lpass-ports.h
+++ b/include/dt-bindings/sound/qcom,q6dsp-lpass-ports.h
@@ -139,6 +139,26 @@
 #define DISPLAY_PORT_RX_5	133
 #define DISPLAY_PORT_RX_6	134
 #define DISPLAY_PORT_RX_7	135
+#define PRIMARY_SDR_MI2S_RX     136
+#define PRIMARY_SDR_MI2S_TX     137
+#define SECONDARY_SDR_MI2S_RX   138
+#define SECONDARY_SDR_MI2S_TX   139
+#define TERTIARY_SDR_MI2S_RX    140
+#define TERTIARY_SDR_MI2S_TX    141
+#define QUATERNARY_SDR_MI2S_RX  142
+#define QUATERNARY_SDR_MI2S_TX  143
+#define QUINARY_SDR_MI2S_RX     144
+#define QUINARY_SDR_MI2S_TX     145
+#define PRIMARY_SDR_TDM_RX_0    146
+#define PRIMARY_SDR_TDM_TX_0    147
+#define SECONDARY_SDR_TDM_RX_0  148
+#define SECONDARY_SDR_TDM_TX_0  149
+#define TERTIARY_SDR_TDM_RX_0   150
+#define TERTIARY_SDR_TDM_TX_0   151
+#define QUATERNARY_SDR_TDM_RX_0 152
+#define QUATERNARY_SDR_TDM_TX_0 153
+#define QUINARY_SDR_TDM_RX_0    154
+#define QUINARY_SDR_TDM_TX_0    155
 
 #define LPASS_CLK_ID_PRI_MI2S_IBIT	1
 #define LPASS_CLK_ID_PRI_MI2S_EBIT	2
-- 
2.34.1

